Dockerfile.fuzz

```diff
@@ -0,0 +1,22 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0-alpine as builder
+
+RUN apk add --no-cache python3 py3-pip bash coreutils
+
+RUN wget https://github.com/microsoft/restler-fuzzer/archive/refs/tags/v9.2.2.tar.gz
+RUN tar -xvf v9.2.2.tar.gz && mv restler-fuzzer-9.2.2 restler-fuzzer
+
+RUN cd restler-fuzzer; mkdir -p restler_bin
+WORKDIR /restler-fuzzer
+RUN python3 ./build-restler.py --dest_dir ./restler_bin/
+COPY fuzzing_docker.sh /restler-fuzzer/fuzzing.sh
+COPY /openapi/v3/* /restler-fuzzer/openapi/
+
+ENTRYPOINT ["/restler-fuzzer/fuzzing.sh"]
+# CMD of "all" will do fuzz-lean test for core-commmand, core-data, core-metadata, support-notifications, and support-scheduler, basically all services under openapi/v3/ directory
+# This can take more than 20 minutes to finish
+CMD ["all"]
+# CMD core-data below only do fuzz-lean for core-data
+# This can take about 2 minutes to finish
+# You may swap out core-data to other server to perform fuzz-lean
+# Comment above CMD of "all" and uncomment below CMD to fuzz core-data only
+# CMD ["core-data", "/restler-fuzzer/openapi/core-data.yaml"]
```

Comment:
I am not wild about `all` being the default. IMO, the `CMD` should not be set in Dockerfile and expected to be passed in when run. This way we have better control and don't accidently run ALL.